src/compiler/AST/Typed.elm

```diff
@@ -196,3 +213,18 @@ recursiveChildren fn ( expr, _ ) =
 
         Tuple3 e1 e2 e3 ->
             fn e1 ++ fn e2 ++ fn e3
+
+
+mapExpr : (a -> b) -> Located ( a, c ) -> Located ( b, c )
+mapExpr =
+    Located.map << Tuple.mapFirst
+
+
+getExpr : Located ( a, b ) -> a
```

Comment:
similar here: `getExpr : LocatedExpr -> Expr_`
I'm a bit torn about whether to name it `getExpr` or `getExpr_` as it doesn't return `Expr` but `Expr_`